churchland_pipeline_python/acquisition.py

```diff
@@ -133,7 +132,7 @@ def populate(self,
 
             # ensure behavior directory exists
             try:
-                if task_controller_hardware == 'Speedgoat':
+                if task & {'task_controller_hardware':'Speedgoat'}:
                     behavior_dir = 'speedgoat'
 
                 next(filter(lambda x: x==behavior_dir, session_files))
```

Comment:
This is an awkward way of checking if the name is in the list. You can use the `in` operator instead.